frb/dm/igm.py

```diff
@@ -342,8 +347,12 @@ def average_DMIGM(z, cosmo = defs.frb_cosmo, f_hot = 0.75, rmax=1., logMmin=10.3
     DM_IGM = DM_cosmic - DM_halos
 
     # Return
-    if cumul:
+    if cumul&return_DMhalos:
+        return DM_IGM, zeval, DM_halos
+    elif cumul&~return_DMhalos:
```

Comment:
I expect these are coded properly, but I get nervous seeing `&~`

can you put parenthesis around and use `not` instead?

Reply:
Done.